assets/js/learners-bulk-actions.js

```diff
@@ -196,15 +197,41 @@ jQuery( document ).ready( function () {
 
 		$modalToggle.attr( 'disabled', true );
 
-		$moreLink.on( 'click', function ( event ) {
-			event.preventDefault();
-			event.stopPropagation();
+		$moreLink.on(
+			'click',
+			function( event ) {
+				event.preventDefault();
+				event.stopPropagation();
+				$( event.target )
+					.addClass( 'hidden' )
+					.prev()
+					.removeClass( 'hidden' );
 
-			$( event.target )
-				.addClass( 'hidden' )
-				.prev()
-				.removeClass( 'hidden' );
-		} );
+				let $userId      = $( event.target ).attr( 'data-user-id' );
+				let $dataNonce   = $( event.target ).attr( 'data-nonce' );
+				let $hiddenPosts = $( event.target ).prev();
+
+				let data = {
+					action: 'get_course_list',
+					user_id: $userId,
+					nonce: $dataNonce,
+				};
+
+				$.ajax(
+					{
+						type: "POST",
+						url: ajax_object.ajax_url,
+						data: data,
+						success:function( data ) {
+							$hiddenPosts.append( data.data );
+						},
+						error: function(errorThrown){
+							$hiddenPosts.append( __( "<p>There was an error fetching courses: ", 'sensei-lms' ) + errorThrown.statusText + ': ' + errorThrown.status + "</p>" )
```

Comment:
To keep it consistent, having the opening/closing tags both outside the translatable string:
```suggestion
							$hiddenPosts.append( "<p>" + __( "There was an error fetching courses: ", 'sensei-lms' ) + errorThrown.statusText + ': ' + errorThrown.status + "</p>" )
```